src/instructlab/helpers.py

```diff
@@ -0,0 +1,369 @@
+# SPDX-License-Identifier: Apache-2.0
```

Comment:
Can you help me understand why these changes are necessary in this PR? 

The commit seems to move the config code, but not remove the old copy.  Also, none of the history from either of the combined files is retained since git does not see any of it as a move.

Reply:
each time I rebase, git (or me) seems to forget config.py. I will fix it on the next rebase or I can redo it